app/src/main/res/layout/progress.xml

```diff
@@ -6,21 +6,9 @@
     android:gravity="center"
     android:orientation="vertical">
 
-    <com.airbnb.lottie.LottieAnimationView
-        android:id="@+id/anim_sketchware"
-        android:layout_width="82dp"
-        android:layout_height="82dp"
-        android:layout_gravity="center"
-        app:lottie_autoPlay="true"
-        app:lottie_fileName="anim/sketchware_logo.json"
-        app:lottie_loop="true" />
+<com.google.android.material.loadingindicator.LoadingIndicator
```

Comment:
When I tested this LoadingIndicator, I set its size to 100dp and it looked nice on my phone, so I think it would look nice here too.
this

Reply:
Ngl it's good and some guys can't see cause they have Myopia and hyperopia or the both at the same time so when you do an exact value for a loading indicator can cause UI issues for some people who make screen too big so it's good and enough and please close this discussion